### PR TITLE
31 branch reports attendance by month

### LINKED ISSUE #31: Branch Reports - Attendance by month.
# REQUIREMENTS
- Add a bar chart of the average time per month
- Add 'average time by month' filter

# DESIGN
Desktop - Branch Reports - Attendance by month

### PR DESCRIPTION
**Pull request by Javi1499**
# Description
what did you implement
- Dynamic rendering of partials based on the selected value
- Chart average hours worked per month using chartkick gem

Why did you implement this?
   - Added dynamic rendering partials to display different types of reports on the same page
   - The graph was used to present information in an easier way.
   - 
  ## Screenshots
If applicable, include screenshots of the results or screenshots that help to see changes
AttendancebyMOnth

### CODE REVIEW

**Review by VictorElizalde — DISMISSED** [review]
LGTM

**Review by juperameza — APPROVED** [review]
LGTM! <3

File: db/migrate/20220325221841_devise_create_admins.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
+
+      ## Lockable
+      # t.integer  :failed_attempts, default: 0, null: false # Only if lock strategy is :failed_attempts
+      # t.string   :unlock_token # Only if unlock strategy is :email or :both
+      # t.datetime :locked_at
+
```
Comment (germanruzca) [inline db/migrate/20220325221841_devise_create_admins.rb:37]: You should delete code that is no needed

**Review by germanruzca — APPROVED** [review]
Just for the comment lines, but LGTM

**Review by femm34 — APPROVED** [review]
LGTM

**Pull request merged**